src/audio/kpb.c

```diff
@@ -1729,7 +1707,6 @@ static void kpb_init_draining(struct comp_dev *dev, struct kpb_client *cli)
 static enum task_state kpb_draining_task(void *arg)
 {
 	struct draining_data *draining_data = (struct draining_data *)arg;
-	struct comp_buffer *sink = buffer_acquire(draining_data->sink);
```

Comment:
I'd keep the pointer - it's used often enough and it would reduce the diff

Reply:
Ok, kept it.